src/pathfora.js

```diff
@@ -1244,6 +1245,21 @@
       } else {
         // NOTE Cannot find Lytics tag, reporting disabled
       }
+
+      if (typeof ga === 'function') {
+        ga(
+          'send',
+          'event',
+          'Lytics',
+          data['pf-widget-action'] || data['pf-widget-event'],
+          '',
+          {
+            nonInteraction: true
+          }
+        );
+      } else {
+        // FUTURE Add exception
```

Comment:
@mjurczyk PR seems fine. You can merge it into develop after adding this exception